fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -1359,13 +1359,30 @@ public ParseNode visitScalarSubquery(StarRocksParser.ScalarSubqueryContext conte
 
     @Override
     public ParseNode visitShowDatabasesStatement(StarRocksParser.ShowDatabasesStatementContext context) {
-        if (context.pattern != null) {
-            StringLiteral stringLiteral = (StringLiteral) visit(context.pattern);
-            return new ShowDbStmt(stringLiteral.getValue());
-        } else if (context.expression() != null) {
-            return new ShowDbStmt(null, (Expr) visit(context.expression()));
+        String catalog = null;
+        if (context.db != null) {
+            QualifiedName qualifiedName = getQualifiedName(context.db);
```

Comment:
actually this a little confused to me, QualifiedNameContext named as db. but we get catalog from this db variable.
how about renaming the QualifiedNameContext variable name of ShowDatabasesStatementContext

Reply:
> 

refactored.`qualifiedName` name now is changed to `dbName`.